linux/system/cpu_frequency.sls

```diff
@@ -0,0 +1,10 @@
+{%- from "linux/map.jinja" import system with context %}
+{%- if system.cpu_frequency is defined %}
+
+ondemand_service_disable:
```

Comment:
Yeap. At the beginning I wrote the formula with cpufrequtils. But its not necessary and even in some servers not compatible (I tested it on Xenial). So its useless. Disabling the daemon is enough.